.github/workflows/rust.yml

```diff
@@ -44,9 +67,8 @@ jobs:
         gst-inspect-1.0 | grep Total
     - name: Build
       run: cargo build
-    - name: Examples
-      run: |
-        ls examples/*.rs | xargs -I{} basename  {} .rs  | grep -v params_connect | RUST_BACKTRACE=1 GST_DEBUG=3 xargs -I{} cargo ex {} --all-features
```

Comment:
Why aren't we running the examples anymore?

Reply:
feel free to r=me to fix the immediate problem though

Reply:
At least for me, one of these examples did not terminate on my local machine. Perhaps I didn't wait long enough for that to happen though. I'll play around a bit locally.

Reply:
It seems this works on CI while running the examples, so I have removed that comit.